index.html

```diff
@@ -1,20 +1,197 @@
+                <div class="photo-wrapper row">
+                    <div class="offset-3 col-6">
+                        <img src="img/Benny_photo.png" alt="Ben's photo">
+                    </div>
+                </div>
+                <div class="name-wrapper row">
+                    <div class="offset-4 col-5">
+                        <h1> Benyam Worku</h1>
+                    </div>
+                </div>
+                <div class=" row">
+                    <div class="separator-line offset-5 col-2">
+                        <!-- blue line -->
+                    </div>
+                </div>
+                <div class="buttons row">
+                    <div class="resume-btn   col-6">
+                        <!-- <button>RESUME</button> -->
+                        <a href="resume.html">
+                            <button type="button" class="btn btn-primary">
+                                RESUME
+                            </button>
+                        </a>
+
+                    </div>
+                    <div class="projects-btn col-6">
+                        <a href="projects.html">
+                            <button type="button" class="btn btn-primary">
+                                PROJECTS
+                            </button>
+                        </a>
+
+                        <!-- <button><a href="projects.html">PROJECTS</a></button> -->
+                    </div>
+                </div>
+                <div class="social-media d-flex justify-content-center align-items-center row">
+                    <a href="#" class="col-2 fa fa-facebook"></a>
+                    <a href="#" class="col-2 fa fa-twitter"></a>
+                    <a href="#" class="col-2 fa fa-linkedin"></a>
+                    <a href="#" class="col-2 fa fa-instagram"></a>
+
+                </div>
+            </div>
+        
+        <!-- Intro text -->
+            <div class="intro-wrapper container" >
+                <div class="row ">
+                    <div class=" offset-1 col-10">
+                        <h1><b>Hello</b></h1>
+
+                        <p>Here's who I am & what I do</p>
+                        <p>
+                            I'm a paragraph. Click here to add
+                            your own text and edit me. It’s easy.
+                            Just click “Edit Text” or double click me to
+                            add your own content and make changes
+                            to the font.</p>
+
+                        <p>I’m a great place for you to
+                            tell a story and let your
+                            users know a little more
+                            about you.</p>
+                    </div>
+                </div>
+            </div>
+        <!-- Footer -->
+            <div id="footer-wrapper">
+                <footer>
+                    <!-- Contacts container -->
+                    <div class="contact container">
```

Comment:
you use 2 different containers here for Call/Write and Social Media. However, this makes the components not aligned into one row, as per the original design (all components need to be in one row). So what you actually need is one container, containing one row. This row will contain Copyright, Call, Write, and Social Media icons. The width for Copyright is wider than the rest, and this can be easily changed as per the doc https://getbootstrap.com/docs/4.0/layout/grid/#variable-width-content